consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -341,13 +341,19 @@ impl<E: Clock, S: Scheme, D: Digest> Round<E, S, D> {
         Some(finalization)
     }
 
-    /// Returns whether at least one honest participant has notarized a proposal.
-    pub fn at_least_one_honest(&self) -> Option<View> {
-        if self.notarizes.len() <= self.scheme.participants().max_faults() as usize {
-            return None;
+    /// Returns `true` if at least one honest participant has notarized a proposal in this view.
+    pub fn at_least_one_honest_notarization(&self) -> bool {
+        if self.finalization.is_some() {
```

Comment:
I didn't think it was worthwhile to handle these cases because fetching anything missing would no longer be necessary?

This was just meant to catch a tip case where we were only seeing `nullifications` (but there was some support for a `proposal` we couldn't vote for and the network couldn't make progress).